src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.js

```diff
@@ -49,7 +71,7 @@ const fetchMarkup = async url => {
   }
 };
 
-const convertInclude = async includeBlock => {
+const convertInclude = async (includeBlock, ...restParams) => {
```

Comment:
Unfortunately this spread operator introduces unused parameters in this function. The reason for this is because the `parseBlockByType()` which invokes this is used generally for all block type conversions and adding the extra `pathname` parameter would be impossible to get on it's own, unless we simulate named parameters with object literals. This would mean a refactor of all block types, this should be considered in this issue https://github.com/bbc/simorgh/issues/6784 which will most likely refactor everything within `getInitialData` to make `pathname` available for every function without having to pass the pathname prop to every single function.

Reply:
One way you could make this slightly more readable, is to assign the pathname to a const at beginning of this function:
`const pathname = restParams[2];`
I'd perhaps also add an explanatory comment above it referencing the issue above, just make clear why you have to do this.